src/core/public/chrome/README.md

```diff
@@ -0,0 +1,127 @@
+
+##    Chrome Service
+
+- [About!](#about-)
+- [Nav Controls Service](#navcontrolsservice-)
+- [Nav Links Service](#navlinksservice-)
+- [Recently Accessed Service](#recentlyaccessedservice-)
+- [Doc Title Service](#doctitleservice-)
+- [UI](#ui-)
+
+## About :
+- **Signature** - `export interface ChromeStart`
+
+The chrome service is a high level UI service that is part of CoreStart (Core services exposed to the Plugin start lifecycle) and offers other plugins a way to add navigation controls to the UI, edit the document title, manipuate navlinks on global header as well as edit the Recent accessed tab. it consists of these sub-services/components.
```

Comment:
This part looks like an overall intro to me.
```
The chrome service is a high level UI service that is part of CoreStart (Core services exposed to the Plugin start lifecycle) and offers other plugins a way to add navigation controls to the UI, edit the document title, manipuate navlinks on global header as well as edit the Recent accessed tab. it consists of these sub-services/components.
```

Right now, display this intro in between `Signature` and `NavControlsService` looks really strange to me. 
<img width="1153" alt="Screen Shot 2022-07-15 at 15 04 42" src="https://user-images.githubusercontent.com/79961084/179317261-dd8eb180-cd16-4f5f-8ada-9908e540ed96.png">

what do you think?

Reply:
Nit: 
`Recent` --> `recent`
`it consists` --> `It consists`